synapse_net/training/domain_adaptation.py

```diff
@@ -109,24 +294,29 @@ def mean_teacher_adaptation(
         if os.path.isdir(source_checkpoint):
             model = torch_em.util.load_model(source_checkpoint)
         else:
-            model = torch.load(source_checkpoint, weights_only=False)
+            model = torch.load(source_checkpoint)
```

Comment:
```suggestion
            model = torch.load(source_checkpoint, weights_only=False)
```

Reply:
We need to keep `weights_only=False` here to be compatible with recent PyTorch versions. If you removed this to fix an error in PyTorch then please update your PyTorch.